packages/terra-theme-context/src/ThemeContextProvider.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ThemeContext from './ThemeContext';
+
+const propTypes = {
+  children: PropTypes.element.isRequired,
+  theme: PropTypes.string,
```

Comment:
These are pretty self explanatory but for consistency can we add some documentation comments?

```jsx
const propTypes = {
  /**
   *
   */
  children: PropTypes.element.isRequired,
  /**
   *
   */
  theme: PropTypes.string,
};
```

Reply:
Updated: https://github.com/cerner/terra-framework/pull/1087/commits/8398f7e62234b6c44502add564933a56987e3a40